### DISCUSSION TITLE
Option to turn off specific mouse buttons so other apps can take ownership [Moved to Issue #355]

### DISCUSSION THREAD

**Discussion by b-ongo** [noah-nuebling/mac-mouse-fix#305] [Other]
For instance I need to customize the forward and back button with different keyboard commands for certain apps (like Apollo) that Mac mouse fix doesn't work with. 

Bringing in a secondary  app like BetterTouchTool would allow me to solve this, but since Mac Mouse Fix takes priority ownership over these buttons I can't do that. Being able to turn off MMF taking over buttons 4 & 5 would be a fantastic option.

**Comment by noah-nuebling** [comment]
Hi! Moving this topic to Issue #355. Please continue any discussion there. Thanks!
